app/endpoints/calendar.py

```diff
@@ -228,6 +228,25 @@ async def delete_bookings_id(
         )
 
 
+@router.post(
+    "/calendar/ical",
+    status_code=204,
+    tags=[Tags.calendar],
+)
+async def create_product(
```

Comment:
The name could be a bit more explicit, like:
```suggestion
async def recreate_ical_file(
```